chrome-extension/content.css

```diff
@@ -1,16 +1,17 @@
-.injection-sidebar {
+#extension-sidebar {
   z-index: 9999;
   position: fixed;
-  right: 0;
-  top: 20px;
+  right: 0px;
+  top: 0px;
   width: 300px;
   min-height: 500px;
-  max-height: 80vh;
+  max-height: 95vh;
+  overflow: scroll;
```

Comment:
Not sure if you intentionally picked this over `overflow: auto` but scroll means that the scroll bar will always be visible even if there is no overflow. auto means it only adds the scroll bar if we need it.

Reply:
ohh thank you!